extras/dev-docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM kshlm/vagrant-centos:latest
+
+MAINTAINER Kaushal M <kshlmster@gmail.com>
+
+ADD ./setup.sh /setup.sh
+RUN chmod +x /setup.sh
+RUN /setup.sh
+RUN rm /setup.sh
+
```

Comment:
Ok. No CMD line here. This makes sense though, as you use it in vagrant + docker + ssh.